src/applications/check-in/components/tests/WhatToDoNext.unit.spec.jsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { expect } from 'chai';
+import { fireEvent, render } from '@testing-library/react';
+import sinon from 'sinon';
+import CheckInProvider from '../../tests/unit/utils/CheckInProvider';
+import {
+  multipleAppointments,
+  singleAppointment,
+} from '../../tests/unit/mocks/mock-appointments';
+import WhatToDoNext from '../WhatToDoNext';
+
+const mockRouter = {
+  currentPage: '/health-care/appointment-check-in/appointments',
+};
+
+const mockStore = {
+  app: 'dayOf',
+};
+
+describe('unified check-in experience', () => {
+  describe('WhatToDoNext', () => {
+    it('displays the what next header', () => {
+      const { getByTestId } = render(
+        <CheckInProvider store={mockStore}>
+          <WhatToDoNext
+            router={mockRouter}
+            appointments={multipleAppointments}
+          />
+        </CheckInProvider>,
+      );
+      expect(getByTestId('what-next-header')).to.exist;
+    });
+    it('displays one card for a single eligible appointment', () => {
+      const { getByTestId } = render(
+        <CheckInProvider store={mockStore}>
+          <WhatToDoNext router={mockRouter} appointments={singleAppointment} />
+        </CheckInProvider>,
+      );
+      expect(getByTestId('what-next-card')).to.exist;
```

Comment:
I would expect this to use `getAllByTestId` and then count that there is only 1 instance. I think this technically works though since it would bark if there were more than one.